test/powershell/Language/Operators/RangeOperator.Tests.ps1

```diff
@@ -0,0 +1,135 @@
+Describe "Range Operator" -Tags CI {
+    Context "Range integer operations" {
+        It "Range operator generates arrays of integers" {
+            $Range = 5..8
+            $Range.count | Should Be 4
+            $Range[0] | Should BeOfType [int]
+            $Range[1] | Should BeOfType [int]
+            $Range[2] | Should BeOfType [int]
+            $Range[3] | Should BeOfType [int]
+        }
+
+        It "Range operator accepts negative integer values" {
+            {
+                -8..-5
+            } |Should Not Throw
+
+            $Range = -8..-5            
+            $Range.count | Should Be 4
+            $Range[0] |Should Be -8
+            $Range[1] |Should Be -7
+            $Range[2] |Should Be -6
+            $Range[3] |Should Be -5
+        }
+
+        It "Range operator support single-item sequences" {
+            {
+                0..0
+            } |Should Not Throw
+
+            $Range = 0..0
+            $Range.count | Should Be 1
+            $Range[0] | Should BeOfType [int]
+            $Range[0] | Should Be 0
+        }
+
+        It "Range operator works in ascending and descending order" {

```

Comment:
Previous tests already test "ascending". I think we could test here only descending order for positive and negative numbers. But I believe we should replace some tests with one - "-2 .. 2" and "2 .. -2" - so we check transition through zero too.